BCnEnc.Net/Shared/DdsFile.cs

```diff
@@ -738,7 +755,12 @@ public enum DxgiFormat : uint
 		DxgiFormatP208,
 		DxgiFormatV208,
 		DxgiFormatV408,
-		DxgiFormatForceUint
+		DxgiFormatForceUint,
+
+		// Added here due to missing documentation of an official value
```

Comment:
Might wanna put these higher up in the enum range. Incase they overwrite something already existing.

Reply:
I will put them at value 300, that gives us a buffer of 181 between ATC and ForceUint.